app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,37 @@
+class ProjectsController < ApplicationController
+  
+  def new
+  	if is_professor?
+  		#@professor = current_professor
+  		@students = Student.all
+  		@project = Project.new
+  	else
+  		redirect_to "/"
+  	end
+  end
+
+  def create
+  	if is_professor?
+	  	new_project = Project.new(title: params[:project][:title], hypothesis: params[:project][:hypothesis], summary: params[:project][:summary], time_budget: params[:project][:time_budget], professor_id: current_user.id)

```

Comment:
please change to rails permit project_params as we discussed, approving to master